helix-sitemap.yaml

```diff
@@ -0,0 +1,12 @@
+sitemaps:
+   website:
+     origin: https://www.zeiss.com
```

Comment:
Using `https://www.zeiss.com` as there's just one origin we can provide in the definition. Likely we should be able to handle this in the CDN by mapping
https://www.zeiss.com to https://www.zeiss.com/en
https://www.zeiss.de to https://www.zeiss.com/de